pkg/gitreceive/build.go

```diff
@@ -11,8 +11,11 @@ import (
 	"time"
```

Comment:
Some of these changes are unrelated to integrating makeup... is it required to tie these changes together?